src/sql/workbench/common/sqlWorkbenchUtils.ts

```diff
@@ -17,5 +17,5 @@ export function getSqlConfigSection(workspaceConfigService: IConfigurationServic
 
 export function getSqlConfigValue<T>(workspaceConfigService: IConfigurationService, configName: string): T {
 	let config = workspaceConfigService.getValue<{ [key: string]: any }>(ConnectionConstants.sqlConfigSectionName);
-	return config[configName];
+	return config ? config[configName] : {};
```

Comment:
Any reason to explicitly return `{}`as opposed to directly returning undefined?

Reply:
this was added for the unit tests.  I'll see if undefined also works fine in the test.